charts/reform-scan-blob-router/values.yaml

```diff
@@ -22,6 +22,8 @@ java:
     FLYWAY_URL: jdbc:postgresql://blob_router-{{ .Values.global.environment }}.postgres.database.azure.com:5432/bulk_scan?sslmode=require
     FLYWAY_USER: blob_router@blob_router-{{ .Values.global.environment }}
     FLYWAY_SKIP_MIGRATIONS: true
+    #Change for Prod, wait required time to get blob
+    STORAGE_BLOB_PROCESSING_DELAY_IN_MINUTES: 0
```

Comment:
maybe it would be safer to use actual value here and overwrite it on preview and aat?

Reply:
yeah good suggestion as in case we forget to override it will apply default which is 30 minutes

Reply:
agree, I thought the same but may be overriding in flux would be safer as any changes here would not effect prod.

Reply:
any changes here will be applied as default unless overriden in flux or env specific  yaml config.

Reply:
that is why I say may be it is better to override in flux prod.